frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -2332,9 +2334,27 @@ func toPBLocation(sourceIndex int, location []parser.Range) pb.Location {
 	}
 }
 
-func reportUnusedFromArgs(args shell.EnvGetter, unmatched map[string]struct{}, location []parser.Range, lint *linter.Linter) {
+func unusedFromArgsTestKeys(env shell.EnvGetter, args map[string]argInfo) map[string]struct{} {
```

Comment:
why "Test" ?

Reply:
The word `Check` slipped my mind, tbh! Updated!